packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -0,0 +1,263 @@
+/* global browser, before, Terra */
+
+describe('Date Picker', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-default'));
+
+    it('creates a hidden input and empty value attr. when no date is entered', () => {
+      expect(browser.getAttribute('[data-terra-date-input-hidden]', 'name')).to.equal('date-input');
+      expect(browser.getAttribute('[data-terra-date-input-hidden]', 'value')).to.equal('');
+    });
+
+    it('sets the date', () => {
+      browser.setValue('input[name="terra-date-date-input"]', '06/01/2017');
+      browser.keys('Enter');
+      browser.click('[class*="button"]');
+    });
+
+    Terra.should.matchScreenshot({ selector: '[class="react-datepicker"]' });
+    /* Terra.should.beAccessible(); */
```

Comment:
Why not run accessibility tests?

Reply:
Accessibility issues have been logged for date picker #1068

Reply:
I will add exceptions for the known issues and run the accessibility tests anyway